nautobot/dcim/filters.py

```diff
@@ -203,24 +286,50 @@ class RackGroupFilterSet(NautobotFilterSet, NameSlugSearchFilterSet):
     )
     parent_id = django_filters.ModelMultipleChoiceFilter(
         queryset=RackGroup.objects.all(),
-        label="Rack group (ID)",
+        label="Parent (ID)",
     )
     parent = django_filters.ModelMultipleChoiceFilter(
         field_name="parent__slug",
         queryset=RackGroup.objects.all(),
         to_field_name="slug",
-        label="Rack group (slug)",
+        label="Parent (slug)",
+    )
```

Comment:
An idea that occurred to me - would it make sense in the case where we have existing `foo` and `foo_id` filters to go ahead and convert the `foo` filter to a `NaturalKeyMultipleChoiceFilter` and document the `foo_id` filter as deprecated? `NaturalKeyMultipleChoiceFilter` should be backwards-compatible with existing usage as a `ModelMultipleChoiceFilter`, and this would begin to provide a migration path for users to begin to move away from `foo_id`.

Reply:
Do you think we should add a deprecation notice in the label?
```py
    parent_id = django_filters.ModelMultipleChoiceFilter(
        queryset=RackGroup.objects.all(),
        label="Parent (ID) - Deprecated (use parent filter)",
    )
```

Reply:
Yeah, good idea. :-)

Reply:
Tracking in #1916